template/protocol.py

```diff
@@ -15,32 +15,22 @@ class SendSignal(bt.Synapse):
     error_message: str = Field("", title="Error Message", frozen=False)
     validator_hotkey: str = Field("", title="Hotkey set by validator", frozen=False)
     miner_order_uuid: str = Field("", title="Order UUID set by miner", frozen=False)
-    required_hash_fields: List[str] = Field(
-        ["signal"],
-        title="Required Hash Fields",
-        description="A list of fields required for the hash.",
-        frozen=False
-    )
+    computed_body_hash: str = Field("", title="Computed Body Hash", frozen=False)
+SendSignal.required_hash_fields = ["signal"]
```

Comment:
nit--would it be cleaner to make required_hash_fields a classvar `required_hash_fields: ClassVar[List[str]] = ["positions"]`

Reply:
Syned in chat